src/game/UICmdList.h

```diff
@@ -0,0 +1,60 @@
+// UICmdListDlg.h: interface for the CUICmdListDlg class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "N3Base/N3UIBase.h"
+#include "N3Base/N3UIButton.h"
+#include "N3Base/N3UIList.h"
+#include "N3Base/N3UIEdit.h"
+
+//////////////////////////////////////////////////////////////////////
+
+class CUICmdListDlg : public CN3UIBase {
+    bool               m_bDestoyDlgAlive;
+    class CUICmdEdit * m_pUICmdEdit;
+    CN3UIButton *      m_pBtn_cancel;
+    CN3UIList *        m_pList_CmdCat;
+    CN3UIList *        m_pList_Cmds;
+    CN3UIScrollBar *   m_pScroll_Categories;
+    CN3UIScrollBar *   m_pScroll_Commands;
+
+  protected:
+    bool  m_bOpenningNow;
+    bool  m_bClosingNow;
+    float m_fMoveDelta;
+    int   m_iRBtnDownOffs;
+    bool  m_bRBtnProcessing;
+
+    enum iCmd {
```

Comment:
I feel like the naming of this enum leaves something to be desired, and will more than likely unintentionally clash with variable names used within this class implementation. Probably something like `CmdListCategory` or something would would avoid any chance of a clash.